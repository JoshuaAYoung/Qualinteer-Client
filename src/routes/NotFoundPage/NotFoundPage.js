import React, { Component } from 'react';

export default class NotFound extends Component {
  render() {
    return (
      <div>
        <img src='http://placekitten.com/400/500' alt='random cat' />
        <p className='404'>404 ERROR - PAGE NOT FOUND</p>
      </div>
    );
  }
}
