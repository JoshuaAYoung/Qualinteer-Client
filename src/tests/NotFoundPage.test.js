import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFoundPage from '../routes/NotFoundPage/NotFoundPage';

describe('NotFoundPage Component', () => {
  it('NotFoundPage Smoke Test', () => {
    const div = document.createElement('div');

    ReactDOM.render(<NotFoundPage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('NotFoundPage Snapshot Test', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
