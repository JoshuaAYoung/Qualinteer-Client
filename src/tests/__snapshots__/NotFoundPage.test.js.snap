// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Component NotFoundPage Snapshot Test 1`] = `
<div>
  <header
    className="notFoundBanner"
    role="banner"
  >
    <h1
      className="pageHeader"
    >
      404
      <br />
      Page Not Found.
    </h1>
    <p
      className="pageInstructions"
    >
      We can’t find the page you’re looking for. Please check the url.
    </p>
  </header>
  <div
    className="notFoundImageContainer"
  >
    <img
      alt="not found"
      className="notFoundImage"
      src="/assets/not-found.svg"
    />
  </div>
</div>
`;
